### ISSUE TITLE
Validation data

### ISSUE THREAD

**Issue by katie-cathy-hunt** [microsoft/DialoGPT#21]
Hey guys! Great work! I really appreciate it!

After reading the code, I noticed that the training data is from 12/2015 to 11/2017, while the test data is from 03/2018 to 01/2019. 

Does that mean that you guys used 12/2017 to 02/2018 as the validation data?

If not, what did you use for the validation data?

**Comment by dreasysnail** [comment]
Hi, thanks for your interest! Yes, please use 12/2017 to 02/2018 as validation set. However we haven't been able to release the reddit data after 12/2017, but you can use our pipeline to create one.

**Comment by ryanminkiscuteaf** [comment]
What's the percentage of the 12/2017 to 02/2018 data did you use for validation?
